### DISCUSSION TITLE
Does the i18n support this configuration？

### DISCUSSION THREAD

**Discussion by qiYuei** [vuejs/vitepress#4241] [General]
Hello, I am now making a component library, the document needs to support Chinese and English multi-language, I need to write the documentation in the component sibling directory for easy maintenance。

I have set vitepress srcDir to packages

- docs
  - .vitepress
- packages
  - components
    - button
      - button.vue
      - button.us.md
      - button.zh.md

I read the documentation that I need to create a ZH like folder in vitepress's working directory, but I can't do that. I've been working on this all afternoon and I still can't figure it out😭.

I wish someone could help me out.

**Comment by qiYuei** [comment]
Can you give me some advice? thank you 
@brc-dd

**Reply by brc-dd** [reply]
Should be possible using [rewrites](https://vitepress.dev/guide/routing#route-rewrites). You'll need to move .vitepress outside docs for it to work though.

**Reply by qiYuei** [reply]
Thanks for your reply. I try to rewrite according to what you said and move.vitepress from docs, but vitepress cannot work normally and I cannot access the route I need

This is my demo repe. Is there something wrong with my configuration that prevents vitepress from accessing the corresponding route?

[demo](https://github.com/qiYuei/wiki)

this is my route config
![image](https://github.com/user-attachments/assets/1f34bce1-f276-40a6-8be1-5f62871d0052)

I want to be able to access my markdown via this route

`http://localhost:5174/zh/components/button/button.html`

`http://localhost:5174/en/components/button/button.en.html`

@brc-dd  thanks~
